src/reducers/versions.spec.tsx

```diff
@@ -1280,14 +1276,14 @@ describe(__filename, () => {
       expect(diff).toHaveProperty('hunks');
     });
 
-    it('returns null if the array of diffs is empty', () => {
+    it('returns null if diff is falsey', () => {
```

Comment:
Can the diff be undefined at any point?

Reply:
It can't, it's either `null` or an empty object in the worst case.

Reply:
@EnTeQuAk Hmm, I'm assuming it's either `null` or ` valid `diff` object. When could it be an empty object? I wonder if I need to deal with that in our code.

Reply:
If it can be an empty object I definitely need to deal with that. So could you confirm if that is in fact possible?

Reply:
Double-checked, no, you're right, only a valid diff or `null` are valid responses. An empty object won't be the case.

Reply:
Great, thanks Chris!